README.md

```diff
@@ -1,8 +1,58 @@
 # React Native Animation Workshop
-This project contains all the information to implement the different animations of the React Native Workshop in BitsConf 2019.
+Welcome to the workshop! You will be implementing some animations using React Native. Before you do that, we need to get you set up.
 
-# TODO
-* Add setup instructions
-* Include videos or GIFs of the animations to be implemented
-* Include references to the documentation most relevant for this
-* Include link to the slides so the sample code is available
+We **recommend that you use a MAC** to run the workshop or pair up with someone who has a MAC.
```

Comment:
are the instructions for windows machines the same as Mac? (apart from the obvious Xcode download).

Reply:
No. I don't even think that you have homebrew for Windows for instance. Should we update all the installation steps to reflect it is Windows only?

Reply:
I've added a note at the beginning of the doc for Windows users not to rely on Homebrew but the linked pages for each component.